src/steps/findings.ts

```diff
@@ -63,11 +63,11 @@ export async function fetchFindings({
             description: findingProps.description,
             category: findingProps.type_category,
             impact: JSON.stringify(findingProps.impact, null, 2), //required to be a string in J1 Finding
-            severity: findingProps.severity, //required property in J1 Finding
+            severity: JSON.stringify(findingProps.severity), //required property in J1 Finding
```

Comment:
What is the goal of this change? The snapshot reveals that the stored value will be `"low"` (includes the quotes), which will be problematic for `find Finding with severity = 'low'` (would have to be `find Finding with severity = '"low"' I think).

Reply:
My bad. On the new SDK version 6.0.0, this line started failing, and I thought this was a number that needed converting, but it was actually a null from a bad entry in our test account data. Fixed with `severity: findingProps.severity || ""`